test/test_distribution.py

```diff
@@ -499,27 +513,30 @@ def test_mvn_gaussian(batch_shape):
 
 
 def _check_mvn_affine(d1, data):
-    assert isinstance(d1, dist.MultivariateNormal)
+    backend_module = import_module(BACKEND_TO_DISTRIBUTIONS_BACKEND[get_backend()])
+    assert isinstance(d1, backend_module.MultivariateNormal)
     d2 = reinterpret(d1)
     assert issubclass(type(d2), GaussianMixture)
     actual = d2(**data)
     expected = d1(**data)
     assert_close(actual, expected)
 
 
+@pytest.mark.xfail(get_backend() == 'jax', reason='dist_to_funsor for jax backend is not available yet')
```

Comment:
It should be straightforward to get these tests working with the JAX backend.  I'll take care of it in a followup PR that deprecates most of `funsor.pyro.convert`.

Reply:
Thanks!